frontend/demo/src/pages/Inbox/Messenger/MessageList/Avatar.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import './Avatar.scss';
+import airyAvatarImage from '../../../../assets/images/icons/airy_avatar.svg';
+
+type AvatarProps = {
+  avatarUrl?: string;
+  isLastMessage: boolean;
+};
+
+const Avatar = (props: AvatarProps) => {
+  const {avatarUrl, isLastMessage} = props;
+
+  return avatarUrl && isLastMessage ? (
+    <div className={'avatarImage'}>
+      <img src={airyAvatarImage} />
+    </div>
+  ) : (
+    <div className={'noAvatar'}>
+      <img />
+    </div>
+  );
+};
```

Comment:
this would probably be a little more readable with an early return for the last message avatar and an explicit return for the no avatar (which I'd probably extract into a local `const noAvatar = () => ...`)